src/taretto/application_skel/{{cookiecutter.application_name}}/entry_points.txt

```diff
@@ -0,0 +1,7 @@
+[{{cookiecutter.application_name}}.application_collections]
+
+[pytest11]
+{{cookiecutter.application_name}} = {{cookiecutter.application_name}}.pytest_plugin
```

Comment:
pytest plugin all the things is not a good pattern per se, shouldn't be the default